cmd/livepeer/livepeer.go

```diff
@@ -515,7 +519,7 @@ func main() {
 			CleanupInterval: cleanupInterval,
 			TTL:             smTTL,
 			RedeemGas:       redeemGas,
-			SuggestGasPrice: backend.SuggestGasPrice,
+			SuggestGasPrice: client.Backend().SuggestGasPrice,
```

Comment:
This ensures that [tx cost estimation in LocalSenderMonitor](https://github.com/livepeer/go-livepeer/blob/887fb1c4e4b2aa84f1d890a23d9378d22bfe8d70/pm/sendermonitor.go#L382) uses the gas price returned by GasPriceMonitor.